src/main/java/frc/robot/subsystems/ampbar/AmpBarConstants.java

```diff
@@ -20,14 +18,10 @@ public final class AmpBarConstants {
   public static final double kAmpBarAmpVoltage = 16; // 8
   public static final double kAmpBarStowVoltage = -16; // -4
 
-  public static final int kAmpBarMotorID = 10;
-  public static int kNumAmpBarMotors = 1;
-  public static double kAmpBarMotorGearing = 7; // i dont think we need this
-  public static double jKgMetersSquared = 0.1;
-  public static double kAmpBarLength = 5;
-  public static double kAmpBarMinAngleDeg = 0;
-  public static double kAmpBarMaxAngleDeg = 90;
-  public static Double kStallVelocityThreshold = 0.1;
+  public static final double ampPosition = 2.7 / 10;
```

Comment:
nitpick: divide by motorGearing

Reply:
its done on purpose so that if the gearing changes, we just change the gearing variable and don't have to find the position again